apps/ichat-gw/ichat-gw.cxx

```diff
@@ -16,15 +16,6 @@ using namespace std;
 
 #define RESIPROCATE_SUBSYSTEM AppSubsystem::GATEWAY
 
-void sleepSeconds(unsigned int seconds)
```

Comment:
I'm guessing you removed this because there is now a similar method provided in rutil/Time.cxx?